docs/contributing/testing-guidelines/unit-testing.md

```diff
@@ -0,0 +1,151 @@
+# Unit testing
+
+Autoware uses the `ament_cmake` framework to build and run tests. The same framework is also used to analyze the test results.
```

Comment:
@maxime-clem Would it be possible to add an explanation about unit testing and its benefits, similar to what you wrote for the integration testing guidelines page? It would keep the two sets of guidelines consistent, and could also be beneficial to new/junior developers.